docs/products/postgresql/howto/monitor-with-pgwatch2.rst

```diff
@@ -1,7 +1,7 @@
-Monitor PostgreSQL metrics with pgwatch2
-========================================
+Monitor PostgreSQL® metrics with pgwatch2
+==========================================
 
-`pgwatch2`_ is an open source monitoring solution for PostgreSQL, created by CYBERTEC and can be used to monitor instances of Aiven for PostgreSQL collecting key PostgreSQL metrics and also gathering data from a wide range of PostgreSQL extensions.
+`pgwatch2`_ is an open source monitoring solution for PostgreSQL®, created by CYBERTEC and can be used to monitor instances of Aiven for PostgreSQL collecting key PostgreSQL metrics and also gathering data from a wide range of PostgreSQL extensions.
```

Comment:
should we also remove `pgwatch2` from dictionary?

Reply:
It's not in the new dictionary, and how it is handled in the new mechanism will need deciding later on, so I think we shall leave this as-is for now (I'm hoping I can come up with a nicer way of handling "this wants to be code font, but also a link").